lib/wannabe_bool/nil.rb

```diff
@@ -2,6 +2,9 @@ module WannabeBool::Nil
   def to_b
     false
   end
+
+  alias_method 'to_boolean', 'to_b'
+  alias_method 'to_bool', 'to_b'
```

Comment:
hi @prodis, just to clarify on the NotImplementedError, that it should be rails to classes that do not yet have .to_b method, right?

Reply:
@galliani The `NotImplementedError` should be raised inside the `#to_b` in the new module. 

The idea here is using [Template Method design pattern|https://en.wikipedia.org/wiki/Template_method_pattern]. The goal is to avoid that this new module had a dependency with the classes/modules it will be included. In that case, the definition of `#to_b` method.